spec/controllers/application_controller_spec.rb

```diff
@@ -33,7 +33,8 @@ def create
       WebMock.stub_request(:get, /\w/).
         to_raise(Excon::Errors::Timeout.new('Request Timeout'))
       post :create
-      expect(Instrumentation.custom_log_items[:api_request_count]).to eq(1)
+      # 2 because we now catch the retries
+      expect(Instrumentation.custom_log_items[:api_request_count]).to eq(2)
```

Comment:
Here I would expect 1 api request, and the log separately that there was 1 retry. Don't know if that is possible though.

That way we can differentiate between an http request trigerring 2 api calls, from another making 1 api call that gets retried.